docs/topics/wasm/wasm-troubleshooting.md

```diff
@@ -0,0 +1,63 @@
+[//]: # (title: Troubleshooting)
+
+Kotlin/Wasm relies on new [WebAssembly proposals](https://webassembly.org/roadmap/) like garbage collection and 
+exception handling to introduce improvements and new features within WebAssembly. 
+
+However, to ensure these features work properly, you need an environment that supports the new proposals. 
+In some cases, you may need to set up the environment to make it compatible with the proposals.
+
+## Browser versions
+
+To run applications built with Kotlin/Wasm in a browser, you need a browser version supporting the new 
+[WebAssembly garbage collection (WasmGC) feature](https://github.com/WebAssembly/gc). Check if the browser version supports 
+the new WasmGC by default or if you need to make changes to the environment.
+
+### Chrome 
+
+* **For version 119 or later:**
+
+  Works by default.
+
+* **For older versions:**
+
+  1. In your browser, go to `chrome://flags/#enable-webassembly-garbage-collection`.
+  2. Enable **WebAssembly Garbage Collection**.
+  3. Relaunch your browser.
+
+  In addition to updating the browser settings, make sure you use the Kotlin toolchain before 1.9.20.
```

Comment:
```suggestion
  In addition to updating the browser settings, make sure you use a Kotlin version before 1.9.20.
```
I checked with developers and "toolchain" was misleading, they actually meant "compiler" so only the Kotlin version needs to be before 1.9.20.